resharper/resharper-unity/src/CSharp/Daemon/Stages/PerformanceCriticalCodeAnalysis/Highlightings/PerformanceHighlightingAttributeIds.cs

```diff
@@ -1,11 +1,12 @@
-namespace JetBrains.ReSharper.Plugins.Unity.CSharp.Daemon.Stages.PerformanceCriticalCodeAnalysis
+namespace JetBrains.ReSharper.Plugins.Unity.CSharp.Daemon.Stages.PerformanceCriticalCodeAnalysis.Highlightings
 {
     // Note that all attribute IDs should start with "ReSharper Unity " to appear properly in both Rider and ReSharper
     public static class PerformanceHighlightingAttributeIds
     {
         public const string CAMERA_MAIN = "ReSharper Unity Expensive Camera Main Usage";
         public const string COSTLY_METHOD_INVOCATION = "ReSharper Unity Expensive Method Invocation";
         public const string NULL_COMPARISON = "ReSharper Unity Expensive Null Comparison";
-        public const string COSTLY_METHOD_HIGHLIGHTER = "ReSharper Unity Performance Critical Context";
+        public const string PERFORMANCE_CRITICAL_CONTEXT_METHOD_HIGHLIGHTER = "ReSharper Unity Performance Critical Context";
+        public const string COSTLY_METHOD_CONTEXT_HIGHLIGHTER = "ReSharper Unity Performance Critical Line Marker";
```

Comment:
This gets displayed to the user, but not sure what it means - if it's the gutter line marker for the current method, do we need a new highlight attribute, or just hide/show it based on context?